data/public_suffix_list.dat

```diff
@@ -11616,3 +11789,4 @@ za.net
 za.org
 
 // ===END PRIVATE DOMAINS===
+0Looking
```

Comment:
This line looks weird, and when I run the mix task to sync this file it removes this line.  Did you add this by accident?  Can it be removed before I merge?